### PR TITLE
Adapted the call to kano-feedback-cli after usage refactor

### PR DESCRIPTION
**Pull request by radujipa**
This simply updates the call to the binary with the new usage
interface.

Related to: https://github.com/KanoComputing/kano-feedback/pull/85

### CODE REVIEW

**Comment by radujipa** [comment]
[(1) As an engineer, I want to change the error messages for V30DB so I can learn more about what's going on [2]](https://trello.com/c/fVSfXXqG/1747-1-as-an-engineer-i-want-to-change-the-error-messages-for-v30db-so-i-can-learn-more-about-whats-going-on-2)

File: bin/kano-capture-logs [hunk 1]
```diff
@@ -25,8 +28,13 @@ def store_logging_error():
 
 
 def get_logs():
-    rc = os.system("USER=root LOGNAME=root HOME=/root kano-feedback-cli {} >{} 2>&1".format(
-        LOG_DEST, LOG_OUTPUT))
+    rc = os.system(
+        'USER=root LOGNAME=root HOME=/root'
+        ' kano-feedback-cli --output {} --screenshot'
```
Comment (Ealdwulf) [inline bin/kano-capture-logs]: Given this happens at boot, I'm not sure it's worth doing a screenshot - on the other hand it's also pretty harmless.

Comment (radujipa) [reply bin/kano-capture-logs]: It used to be there, but the `--title` tag will be more useful to know where it's coming from. Leaving it there obviously makes it slower to bootup into Safe Mode, and the archive a bit bigger. I'll remove it 👍

File: bin/kano-capture-logs [hunk 2]
```diff
@@ -38,13 +45,16 @@ def get_log_size():
...
+
+    get_logs()
 
-if (get_boot_space()-get_log_size()) > SPACE_MARGIN:
-    os.system("cp {} /boot".format(LOG_DEST))
-    os.system("syncfs </boot/config.txt")
-    play_sound(SOUND_FILE)
+    if (get_boot_space() - get_log_size()) > SPACE_MARGIN:
+        os.system("cp {} /boot".format(LOG_DEST))
+        os.system("syncfs </boot/config.txt")
```
Comment (skarbat) [inline bin/kano-capture-logs]: Is this `syncfs` meant to flush `config.txt` or the logfile?

Comment (Ealdwulf) [reply bin/kano-capture-logs]: I don't know which is intended, but it applies to the whole filesystem so it will do both.

**Pull request merged**